go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/dell/dell-csi-extensions/podmon v1.5.0
 	github.com/dell/dell-csi-extensions/replication v1.8.0
 	github.com/dell/dell-csi-extensions/volumeGroupSnapshot v1.6.0
-	github.com/dell/gobrick v1.11.2
+	github.com/dell/gobrick v0.0.0-20240919095217-08217598cbb4
```

Comment:
Shouldn't this be v**1.11.3**-20240919..... ?
It will be easy to review the change when we move to final v1.11.3 tag later on.

Reply:
That did not work. When we get the final tag we can make the change as we get close to the release.

Reply:
Fixed.